JSQMessagesViewController/Views/JSQMessagesComposerTextView.m

```diff
@@ -156,6 +156,13 @@ - (void)setTextAlignment:(NSTextAlignment)textAlignment
     [self setNeedsDisplay];
 }
 
+- (void)paste:(id)sender
+{
+    if (!self.pasteDelegate || [self.pasteDelegate composerTextView:self shouldPasteWithSender:sender]) {
```

Comment:
That means that if there's no paste delegate, [super paste:sender] won't get called.